src/core/expr/fexpr.cc

```diff
@@ -669,6 +671,69 @@ DECLARE_METHOD(&PyFExpr::first)
     ->docs(doc_first);
 
 
+static const char* doc_as_type =
+R"(as_type()
+--
+
+Equivalent to :func:`dt.as_type(self)`.
+)";
+
+oobj PyFExpr::as_type(const XArgs& args) {
+  auto as_typeFn = oobj::import("datatable", "as_type");
+  oobj n = args[0].to_oobj();
+  return as_typeFn.call({this, n});
+}
+
+
+DECLARE_METHOD(&PyFExpr::as_type)
+    ->name("as_type")
+    ->docs(doc_as_type)
+    ->arg_names({"n"})
+    ->n_positional_or_keyword_args(1);
```

Comment:
Do you want to allow a keyword argument here? For example, does `f[0].as_type(n=int)` makes sense?
When a function takes a single argument only, we more often than not want it to be positional-only.
Also, this argument is required (calling `.as_type()` without args makes no sense), so you should add `->n_required_args(1)`.

Reply:
@st-pasha it throws an error if I set it as required args : `E       datatable.exceptions.TypeError: Method datatable.FExpr.as_type() takes no positional arguments, but 1 was given`

Reply:
I'm not sure what happens here. Can you elaborate please on how exactly you specify `DECLARE_METHOD()...`, and then how you call the method?